changelog.md

```diff
@@ -0,0 +1,132 @@
+
+-   UPWARD spec introduced for cross platform server behavior configuration.
+    An upward spec describes a stack-agnostic server that is used for server side rendering.
+-   A PWA package can now define its network requirements in an UPWARD spec file.
+-   A PWA can now be deployed on any tech stack as long as the server is UPWARD compliant.
+-   UPWARD server implemented in NodeJS and PHP
+-   UPWARD spec published and open for community contribution.
+
+#### Related documentation updates
+
+-   [UPWARD spec overview][]([#382][])
+-   [UPWARD-js server][]([#382][])
+
+### Improved developer experience
+
+-   Implemented consistent patterns for workflow, such as action/reducer organization and container/presentational separation, in the Venia reference theme source code.
+-   Venia components are now portable and can be used in any React application.
+-   New centralized driver and adapter concepts for all PWA-Studio component input/output.
+-   Configurable low level UI elements, such as lists and forms, available for developers without the need for drivers or adapters.
+-   Improved error handling to help detection and recoverability.
+
+#### Related documentation updates
+
+-   [Modular Venia components][] ([#796][])
+-   [Venia directory structure][]([#487][] [#752][])
+
+### App-like experience
+
+-   Pagination feature added to Venia reference storefront for app-like navigation.
+-   Inline checkout implemented in the Venia reference storefront.
+-   Basic account creation and management implemented in Venia.
+-   Loading state implemented to improved the app-like experience.
+
+#### Related documentation updates
+
+-   [Venia Checkout implementation][] ([#673][])
+
+### Improved shopper experience
+
+-   Braintree integration in the Venia reference storefront.
+-   Search with intelligent previews implemented in the Venia reference storefront.
+
+#### Related documentation updates
+
+-   [Braintree integration][]([#793][])
+
+### PWA compliance
+
+-   Web App Manifest file created for the Venia reference storefront.
+    This file provides the metadata for adding the Venia storefront to a mobile home screen.
+-   Service worker optimizations
+
+### Performance enhancements
+
+-   Smart caching implemented to improve service worker catalog caching
+-   Bandwidth usage optimizations achieved using:
+    -   Dynamic image compression/resize
+    -   Advanced minification
+    -   Bundle splitting
+-   Improved app performance on devices using smoother animations and non-blocking renders, which provide faster times on first meaningful paint.
+
+### GraphQL usage
+
+-   GraphQL usage increased to match coverage updates in Magento 2.3.0.
+
+### Magento Cloud compatibility
+
+-   A new Magento module (`Magento_UpwardConnector`) is available to route frontend requests through an UPWARD-PHP server.
+-   Software Integrators(SI) can replace existing Magento cloud storefronts with a new PWA storefront using the UPWARD connector module.
```

Comment:
```suggestion
-   Solusion Integrators(SI) can replace existing Magento cloud storefronts with a new PWA storefront using the UPWARD connector module.
```

I _think_ this is what they want to be called?

Reply:
Did you mean `Solution`?

Reply:
Uh...